packages/situational-awareness-bof.vm/situational-awareness-bof.vm.nuspec

```
@@ -2,12 +2,13 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>situational-awareness-bof.vm</id>
-    <version>0.0.0.20250219</version>
+    <version>0.0.0.20250506</version>
     <authors>trustedsec</authors>
-    <description>Provides a set of basic situational awareness commands implemented in a Beacon Object File (BOF). This allows you to perform some checks on a host before you begin executing commands that may be more invasive.</description>
+    <description>Situational Awareness BOF providessituational awareness commands implemented in a Beacon Object File which allows to perform some checks on a host before executing commands.</description>
```

There is a space missing (`provides situational`). Improvement proposal:
```suggestion
    <description>Situational Awareness BOF offers basic host checks in a Beacon Object File, enabling pre-execution reconnaissance before more invasive commands.</description>
```